src/ILLink.Shared/SharedStrings.resx

```diff
@@ -141,12 +141,24 @@
   <data name="RequiresAssemblyFilesMessage" xml:space="preserve">
     <value>Using member '{0}' which has 'RequiresAssemblyFilesAttribute' can break functionality when embedded in a single-file app.{1}{2}</value>
   </data>
+ <data name="RequiresDynamicCodeTitle" xml:space="preserve">
+    <value>Members annotated with 'RequiresDynamicCodeAttribute' require dynamic access otherwise can break functionality when trimming application code</value>
```

Comment:
```suggestion
    <value>Members annotated with 'RequiresDynamicCodeAttribute' can break functionality when compiling the application ahead of time.</value>
```

Or something like that - the message seems to be more about trimming, which is not the problem in this case. I think we will need somebody like Andy to review this message.

Reply:
Yes, I would like Andy to have a look at this as well so that we don't need to go back and forth about this later.